CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+- Add support for `StencilSwiftKit`'s additional filters. [#1994](https://github.com/tuist/tuist/pull/1994) by [@svastven](https://github.com/svastven).
```

Comment:
It is not PR related but can you add `###Added` section's header?

Reply:
I've already added it in this [PR](https://github.com/tuist/tuist/pull/1993), so I think it's fine to keep it as-is, but good catch 👍 🙂